# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../gens/gcc4.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

diff -Nur GensForLinux-orig/src/gens/emulator/g_main.c GensForLinux/src/gens/emulator/g_main.c
--- GensForLinux-orig/src/gens/emulator/g_main.c	2004-05-18 20:34:00.000000000 +0000
+++ GensForLinux/src/gens/emulator/g_main.c	2006-02-03 21:56:33.828752000 +0000
@@ -5,6 +5,7 @@
 #include <sys/stat.h>
 #include <stdlib.h>
 #include <stdio.h>
+#include <string.h>
 #include "g_main.h"
 #include "timer.h"
 #include "gens.h"
diff -Nur GensForLinux-orig/src/gens/emulator/g_main.h GensForLinux/src/gens/emulator/g_main.h
--- GensForLinux-orig/src/gens/emulator/g_main.h	2004-04-15 20:22:49.000000000 +0000
+++ GensForLinux/src/gens/emulator/g_main.h	2006-02-03 21:58:00.890193000 +0000
@@ -36,6 +36,8 @@
 extern unsigned char Keys[];
 extern unsigned char joystate[];
 
+static int Build_Language_String (void);
+
 void SetWindowText(const char *text);
 
 int Set_Render(int Full, int Num, int Force);
diff -Nur GensForLinux-orig/src/gens/emulator/parse.c GensForLinux/src/gens/emulator/parse.c
--- GensForLinux-orig/src/gens/emulator/parse.c	2004-05-19 19:21:00.000000000 +0000
+++ GensForLinux/src/gens/emulator/parse.c	2006-02-03 21:58:14.467041500 +0000
@@ -1,6 +1,7 @@
 #include <stdlib.h>
 #include <stdio.h>
 #include <getopt.h>
+#include <string.h>
 #include "port.h"
 #include "save.h"
 #include "cpu_68k.h"
